README.mediawiki

```diff
@@ -505,7 +505,7 @@ It is no coincidence that the battlefield of modern democracies is disputed in t
 
 Corporations and public institutions are prone to corruption because decisions often happen in secrecy behind closed doors while accounting happens over time. Effectively, organizations are ''decision laundering'' by disconnecting accountability from the decisions. The lack of an incorruptible timeline storing financial and political decisions enables such inaccountability. The [https://en.wikipedia.org/wiki/Leviathan_(book) ''Leviathan'' State] is an inefficient machine: although it proclaims itself as the sovereign ruler for any given population by means of force, whoever is in charge of running its bureaucracy can still be corrupted making the whole house of cards fall apart. This distance between fact and accounting is the source of gossip.
 
-The building blocks of institutions consist on facts that define agreements. But the kind of facts agreements contain are of a very specific type: Institutions are not built with objective facts that are scientific, measurable and independent from human judgment; but rather inter-subjective facts that build the social world within a community setting the relations of property and rights. For example, the notion that every red can of soda belongs to the Coca Cola Corporation is not objective but an inter-subjective fact agreed upon all members of society acknowledging the intellectual property rights that a company has over its product. In this way, institutional reality helps scale economic relations and reduce the information required for organizations to transact.
+The building blocks of institutions consist of facts that define agreements. But the kind of facts agreements contain are of a very specific type: Institutions are not built with objective facts that are scientific, measurable and independent from human judgment; but rather inter-subjective facts that build the social world within a community setting the relations of property and rights. For example, the notion that every red can of soda belongs to the Coca Cola Corporation is not objective but an inter-subjective fact agreed upon by all members of society acknowledging the intellectual property rights that a company has over its product. In this way, institutional reality helps scale economic relations and reduces the information required for organizations to transact.
```

Comment:
fact agreed upon **by** all members

Wonder why github fails to highlight the added word…